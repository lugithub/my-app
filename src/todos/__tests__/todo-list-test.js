import React from 'react';
import renderer from 'react-test-renderer';

import TodoList from '../todo-list/';

describe('TodoList', () => {
  it('should render todos', () => {
    const component = renderer.create(<TodoList todos={[{id: 1, label: 'a'}, {id: 2, label: 'b'}]}/>);
    expect(component).toMatchSnapshot();
  });
});