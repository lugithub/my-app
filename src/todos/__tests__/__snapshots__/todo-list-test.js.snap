// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoList should render todos 1`] = `
<ul>
  <li>
    a
  </li>
  <li>
    b
  </li>
</ul>
`;
